plugins/fileservices/fileservices.cpp

```diff
@@ -142,6 +142,8 @@ static const char * EclDefinition =
 "  varstring GetEspURL(const varstring username = '', const varstring userPW = '') : c,once,entrypoint='fsGetEspURL'; \n"
 "  varstring GetDefaultDropZone() : c,once,entrypoint='fsGetDefaultDropZone'; \n"
 "  dataset(FsDropZoneRecord) GetDropZones() : c,context,entrypoint='fsGetDropZones'; \n"
+"  integer4 GetExpireDays(const varstring lfn) : c,context,entrypoint='fsGetExpireDays'; \n"
+"  SetExpireDays(const varstring lfn, integer4 expireDays = -1) : c,context,entrypoint='fsSetExpireDays'; \n"
```

Comment:
From the ECL users perspective, it would be clearer when wanting to remove expiry, if there was a ClearExpiryDays(const varstring lfn), and for SetExpiryDays to only accept a value of >= 0

Reply:
Done